.github/workflows/validate-submodule-pull-request.yml

```
@@ -19,14 +18,15 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - name: Prefix PR with issue key
+    - name: Validate Jira Issue Code in Pull Request
```

```suggestion
    - name: Add Jira Issue Code to Pull Request
```
?

It doesn't actually do any validation. It won't fail if the issue code is not there.